_includes/apple-touch-icons.md

```diff
@@ -0,0 +1,9 @@
+{% comment -%}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Could not find module `remark-inline-links`](https://app.codacy.com/gh/SwedbankPay/swedbank-pay-design-guide-jekyll-theme/pullRequest?prid=5634671)